README.md

```diff
@@ -0,0 +1,40 @@
+# Getting Started
```

Comment:
This readme doesn't tell me what this vagrant box does. What does it do? Why should I use it? How is it going to make my life easier?